Orm/Xtensive.Orm/Orm/Building/SystemModule.cs

```diff
@@ -46,7 +46,7 @@ private static void ProcessRecycledDefinitions(DomainBuilderConfiguration config
 
     private static void ApplyModelFixesForMySql(DomainModelDef model)
     {
-      BuildLog.Info("Applying changes to Metadata-related types for MySQL");
+      BuildLog.Info(nameof(Strings.ApplyingChangesToMetadata));
```

Comment:
For logging messages we use "Log" prefix, you can find examples across this PR changes. Also this name of property give no clue about relation to MySql. As a general rule we remove spaces and transform it to PascalCase like so
```
"Applying changes to Metadata-related types for MySQL" -> Strings.LogApplyingChangesToMetadataRelatedTypesForMySQL
```
If phrase is too long it's a hint to optimize it if possible, otherwise, we optimize resource name to be short but still reflect the meaning of entire message. Here I'd suggest ```LogApplyingChangesToMetadataTypesForMySQL``` as name for resource string.

Reply:
done